README.md

```diff
@@ -30,14 +30,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+The DOM stands for Document Object Model. The HTML is compiled by the browser and creates the DOM so the developer can manipulate objects.
+
```

Comment:
Correct. The main thing to remember is the DOM is a representation of the document.